rosco-web/pkg_scripts/postInstall.sh

```diff
@@ -14,16 +14,17 @@ fi
 # install packer
 function install_packer() {
   PACKER_VERSION="0.12.1"
+  TEMPDIR=$(mktemp -d installrosco.XXXX)
   local packer_version=$(packer --version)
   local packer_status=$?
   if [[ $packer_status -ne 0 ]] || [[ "$packer_version" != "$PACKER_VERSION" ]]; then
     pushd .
-    cd /usr/bin
+    cd $TEMPDIR
     wget https://releases.hashicorp.com/packer/${PACKER_VERSION}/packer_${PACKER_VERSION}_linux_amd64.zip
     apt-get install unzip -y
-    unzip -o "packer_${PACKER_VERSION}_linux_amd64.zip"
-    rm "packer_${PACKER_VERSION}_linux_amd64.zip"
+    unzip -o "packer_${PACKER_VERSION}_linux_amd64.zip" -d /usr/bin
     popd
+    rm -rf $TEMPDIR
```

Comment:
This should be outside the `if`, so the $TEMPDIR is deleted even when packer is already installed. (Or move the `mktemp` call into the if statement).